governance/polls/SparkLend Ethereum - Increase ETH borrow cap gap parameter to 20k - March 25, 2024.md

```diff
@@ -0,0 +1,55 @@
+---
+title: SparkLend Ethereum - Increase ETH borrow cap gap parameter to 20k - March 25, 2024
+summary: Signal your support or opposition to increasing the ETH borrow cap gap parameter to 20k ETH
+discussion_link: https://forum.makerdao.com/t/mar-21-2024-proposed-changes-to-sparklend-for-upcoming-spell/23918
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-03-25T16:00:00
+end_date: 2024-03-28T16:00:00
+---
+# Poll: SparkLend Ethereum - Increase ETH borrow cap gap parameter to 20k - March 25, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, March 25 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following SparkLend [CapAutomator](https://github.com/marsfoundation/sparklend-cap-automator) change:
+
+- Increase [ETH Market](https://app.spark.fi/reserve-overview/?underlyingAsset=0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2&marketName=proto_spark_v3) borrow cap `gap` parameter by 10,000 ETH from 10,000 ETH to **20k ETH**
```

Comment:
:yellow_circle: **minor**:

Consistency with units, see comment above. However, if you disagree and you want to use `k`, make all values `k`.

```suggestion
- Increase [ETH Market](https://app.spark.fi/reserve-overview/?underlyingAsset=0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2&marketName=proto_spark_v3) borrow cap `gap` parameter by 10,000 ETH from 10,000 ETH to **20,000 ETH**
```